src/trace-plugin-loader.js

```diff
@@ -95,6 +92,11 @@ function activate(agent) {
         }
         instrumentation.patches[moduleRoot] = patchSet;
       }
+
+      // Create a new object exposing functions to create trace spans and
+      // propagate context. This relies on functions currently exposed by the
+      // agent.
+      var api = new PluginAPI(agent, moduleRoot);
```

Comment:
`moduleRoot` is not the correct value to use here. It should be the name of the module, but `moduleRoot` is the path to the module. In the case of `http` it is `null`.

Unfortunately, the name of the module is not accessible in this scope, so you'll have to add a `name: moduleName` field in `plugins[moduleName]` at the top of this function. Then you can use `instrumentation.moduleName`.